frontend/src/styles/Lectern.css

```diff
@@ -20,10 +20,32 @@
 }
 
 .lectern .runningHead div {
-  padding-top: 17px;
+  padding-top: 5px;
   padding-bottom: 17px;
 }
 
+.lectern .runningHead .info-container {
+  padding-bottom: 0px;
+  display: block;
+  text-align: right;
+}
+
+.info-creation-document {
+  position: relative;
+  border: none;
+  background: none;
+  color: crimson;
+}
+.text-document-creation {
+    display: none;
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    transform: translateX(-50%);
+    z-index: 10;
+    white-space: nowrap;
+}
+
```

Comment:
Because the corresponding elements are in `OpenedDocuments` instead of `Lectern`, you may create a distinct stylesheet.